doc/developer-notes.md

```diff
@@ -171,3 +171,36 @@ Threads
 - BitcoinMiner : Generates bitcoins (if wallet is enabled).
 
 - Shutdown : Does an orderly shutdown of everything.
+
+Ignoring IDE/editor files
+--------------------------
+
+In closed-source environments in which everyone uses the same IDE it is common
+to add temporary files it produces to the project-wide `.gitignore` file.
```

Comment:
In my experience in - much more heterogeneous, multiplatform - open source projects it's rare to see ignores for specific tools that some of the developers use